biopandas/pdb/pandas_pdb.py

```diff
@@ -864,3 +864,29 @@ def to_pdb_stream(self, records: tuple[str] = ("ATOM", "HETATM")) -> StringIO:
         output.write("\n")
         output.seek(0)
         return output
+
+    @staticmethod
+    def gyradius(df: pd.DataFrame) -> float:
```

Comment:
Could this default to `self.df["ATOM"]`?

`ppdb.gyradius(ppdb.df["ATOM"]` seems a little clunky

Reply:
E.g. 

`if df is None: df = self.df["ATOM"]`

Reply:
Yeah, I thought that it might be better, but I just imitated the usage of `rmsd`, but I guess in its case you might want to use it for other atoms than just the ones in the residues? But yes I can fix that.

Reply:
Should I just remove the `df` parameter altogether and set `df = self.df['ATOM']`? Because with anything else other than 'ATOM' it won't work. 'ANISOU' and 'OTHERS' don't have coordinates, and 'HETATM' has atoms that are most likely not present in `atomic_masses`, and I don't know if anyone would even want to use the function for 'HETATM'. It could be made as a parameter that defaults to 'ATOM', but again, anything else doesn't really make sense

Reply:
So I think the way I would do it is to implement a standalone function and then have a method in the class that wraps that and calls it on self.df["ATOM"]. Having said that, I can imagine an edge case scenario where someone wants to compute gyradius on HETATMs (they're not always strictly non-protein atoms in pactice!)

Reply:
So I eventually added a `records` parameter so now you can specify if you want the rg for the "ATOM" record or "HETATM" or both. Not sure if atoms in HETATM would be relevant, but if they are for someone, more atom masses can be added to `atomic_masses`. There's also a `decimals` parameter that controls the rounding which is a a bit useless since anyone could probably use `round()`, but the final value had 14 decimals, so I thought that 4 is a good default for rounding and gives the option to change it.

Reply:
Awesome, final suggestion then is to move the atomic masses into `constants.py` (see #119 ). That way it's easy to accomodate hetatms like you suggest. 

I.e. 

```python
import biopandas

biopandas.ATOMIC_MASSES = {"Si": 100, "Mg: 200"}

# Compute gyraidus...
```

Reply:
How do I add them to `constats.py` considering that you've added the file in your recent pull request? Do I have to clone your main fork and then make a PR, and once merged the changes will be included in your current PR to biopandas?

Reply:
If you just make the file on this branch I think it should merge automatically. Otherwise I'll resolve the conflict on my branch